helm-chart/metabase/templates/deployment.yaml

```diff
@@ -89,16 +89,32 @@ spec:
               secretKeyRef:
                 name: {{ include "metabase.fullname" . }}-secrets
                 key: MB_DB_HOST
+          {{- if .Values.secrets.existingSecret.existingSecretPasswordKey }}
+          - name: MB_DB_PASS
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.secrets.existingSecret.existingSecretName }}-existingSecrets
```

Comment:
This (and the all other instances where `-existingSecrets` is present) should be:

```yaml
name: {{ .Values.secrets.existingSecret.existingSecretName }}
```

WDYT about providing a default name in case `existingSecretName` is not set?

Reply:
sorry, didn't understand "WDYT about providing a default name in case existingSecretName is not set?"

Reply:
@paoliniluis I was thinking something along the lines of

```yaml
name: {{ or .Values.database.existingSecret.name (printf "%s-secret" (include "metabase.fullname" .)) }}
```

Reply:
Thanks, but according to what I'm reading is that this feature is specifically for deployments that have already a secret configured, so either way you have to enter a secret name (a specific one which is already created), or there's no secret (so the section in the yaml file won't be written). Is this ok?

Reply:
Makes sense :+1: